tests/StacktraceTest.php

```diff
@@ -203,7 +203,7 @@ public function buildFromBacktraceDataProvider(): \Generator
             ],
         ];
 
-        yield 'Backtrace with frame containing memory address' => [
+        yield 'Backtrace with frame containing memory address 0x format' => [
```

Comment:
What about changing the description to something more meaningful, e.g. `Backtrace with frame containing memory address in PHP <7.4.2 format`?